mongodb/leafygreen-ui
# Shared tooling: Storybook `@lg-tools/storybook`

## ✍️ Proposed changes

- Creates `@lg-tools/storybook` for storybook config
- Creates `@lg-tools/storybook-decorators` for storybook decorators

Separates these two packages so we can use any @lgui component within a decorator, without worrying about circular dependencies.

Not included in this PR:
- Moving code from `lib/storybook`

## 🧪 How to test changes

- Ensure Storybook builds
- Ensure Chromatic runs
- Ensure website Live Examples look right
